lib/src/ovis_log/ovis_log.c

```diff
@@ -527,7 +533,7 @@ char *ovis_log_list(const char *subsys)
 			goto err;
 		}
 	} else {
-		rc = __buf_append(buf, "{\"name\":\"%s\","
+		rc = __buf_append(buf, "{\"name\":\"%s (default)\","
```

Comment:
@nichamon what is the (default) for?

Reply:
It is to specify that this is the default log.